src/data/nat/log.lean

```diff
@@ -64,6 +64,11 @@ begin
   exact if_pos ⟨le_floor hn, h⟩,
 end
 
+/-- A special case of `log_of_one_lt_of_le` for `R = ℕ` -/
+lemma log_of_one_lt_of_le' {b n : ℕ} (h : 1 < b) (hn : b ≤ n) :
+  log b n = log b (n / b) + 1 :=
+log_of_one_lt_of_le h $ (nat.cast_le.mpr hn).trans_eq (nat.cast_id _)
```

Comment:
I suspect this works
```suggestion
log_of_one_lt_of_le h hn
```

Reply:
I lied in the PR description, `nat.cast_id` is not true by rfl